### DISCUSSION TITLE
View text document attached in comment without downloading

### DISCUSSION THREAD

**Discussion by williamcroome** [community/community#64065] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Sometime we attached text files (normally some sort of log) to a comment. 
Currently in GitHub.com when I click on the file it downloads.
Is there anyway to view natively. 
Maybe this is an issue of browser rather than GitHub

Thanks for any advice.

**Comment by WilliamHCarter** [comment]
There is no native way to view attached txt files within github comments to my knowledge, but here are a few things you can try:

**Use Gists**
A [gist](https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists) is a github-friendly way to share code or text snippets, and then display them natively in the browser. You could then simply attach the gist to the comment. This would take a few seconds of extra effort to do each time, but would render in the way you want.

**Use Markdown Details**
Another solution is to use the markdown features of Github, such as the 'details' and 'summary' tags to make a collapsable area that you could paste the logs directly into. This might come with some formatting issues but is an easy and convenient way to show the full logs from a comment.

Here's an example of how that may look:

**code:**
```
    <details>
        <summary> click to expand </summary>
        Here is where your text would go, such as a log.
    </details>
```

**implementation:**
<details>
    <summary> click to expand </summary>
    Here is where your text would go, such as a log.
    </details>

Hope this helps answer your question, let me know if you need any clarification.
